input/fsh/DkCoreBasicParameter.fsh

```diff
@@ -164,21 +164,21 @@ Description: "An example of TOBS (Danish: Tidlig opsporing af begyndende sygdom)
 Usage: #example
 * type = #collection
 * timestamp = "2023-09-12T17:45:00.000Z"
-* entry[+].fullUrl = "Patient/else"
+* entry[+].fullUrl = "http://hl7.dk/fhir/core/StructureDefinition/Patient/else"
```

Comment:
This looks odd

Reply:
"Patient/else" causes this error "The fullUrl must be an absolute URL (not 'Patient/else')" in QA - that's why. Suggestions are welcome.

Reply:
The fullUrl given here leaves the impression that it references a structure definition which is not the case

Reply:
Alright - what do you suggest it is in stead?

Reply:
Either use uuid's or remove the structure definition part of the url

Reply:
so rather e.g. http://hl7.dk/fhir/core/Patient/else or http://example.com/Patient/else or urn:uuid:88f151c0-a954-468a-88bd-5ae15c08e057

Reply:
I'll make sure to correct